auth-web/src/components/auth/create-account/PaymentMethodSelector.vue

```diff
@@ -32,10 +35,11 @@
           data-test="save-button"
           :disabled="!isEnableCreateBtn"
         >
-          Create Account
+         {{ readOnly ? 'Submit' : 'Create Account'}}
```

Comment:
hmm this variable name is confusing. Submit if it's read-only? May be rename it for better readability.

Reply:
added more comments